app/models/ability.rb

```diff
@@ -89,6 +89,16 @@ def signed_in(user)
     can :manage, Commercial, commercialable_type: 'Event', commercialable_id: user.events.pluck(:id)
 
     can [:destroy], Openid
+
+    can [:new, :create], Track do |track|
+      track.new_record? && track.program.cfps.for_tracks.try(:open?)
+    end
+
+    can [:index, :show, :restart, :confirm, :withdraw], Track, submitter_id: user.id
```

Comment:
I should be able to access show action of confirmed tracks, to get their details.